reproductions/MissingLibraryCrash/Directory.Build.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <!-- This file is necessary to prevent automatic including of the managed library for reproduction purposes -->
+
+  <PropertyGroup>
+    <Platforms>x64;x86</Platforms>
+    <PlatformTarget>$(Platform)</PlatformTarget>
+    <IsPackable>false</IsPackable>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <ProfilerOutputDirectory>..\..\src\Datadog.Trace.ClrProfiler.Native\bin\$(Configuration)\$(Platform)</ProfilerOutputDirectory>
```

Comment:
How come you excluded the following lines? If my understanding is correct, that wouldn't included the managed DLL, just the native DLL and the integrations.json file.

```xml
  <ItemGroup>
    <None Include="$(ProfilerOutputDirectory)\*.dll;$(ProfilerOutputDirectory)\*.so;$(ProfilerOutputDirectory)\*.pdb"
          CopyToOutputDirectory="Always"
          CopyToPublishDirectory="Always"
          Link="profiler-lib\%(RecursiveDir)\%(Filename)%(Extension)" />
    <Content Include="..\..\integrations.json"
             CopyToOutputDirectory="Always"
             CopyToPublishDirectory="Always"
             Link="profiler-lib\integrations.json" />
  </ItemGroup>
```

Reply:
I should definitely include the profiler, but I am intentionally using a different integrations.json file.